src/lib/helpers/attendance.ts

```diff
@@ -211,8 +384,9 @@ const calculateWorkData = (
     const hours = graphRecordsInputData.workSummary[key].workHours;
     const data: GraphDataPoint = {
       x: dayMapper[key],
-      y: parseFloat(hours),
+      y: parseFloat(String(hours)),
```

Comment:
```suggestion
      y: hours,
```
check whether hours come as float from the API